exlibs/nant.exlib

```diff
@@ -1,7 +1,7 @@
 # Copyright 2012 Nikolay Orlyuk <virkony@gmail.com>
 # Distributed under the terms of the GNU General Public License v2
 
-export_exlib_phases src_compile
+export_exlib_phases src_compile src_install
```

Comment:
Thank you for taking look into this.

This change looks to do partial revert of f8eb0f6257597ae6ce16bf894c756675f4b0dbb1 where I think `src_install` was left out of exporting because it was not implemented at all while `dotnet.exlib` got implementation for it. I don't remember exactly why I didn't dropped that function.

Do you think there might be a benefit now to have `nant_src_install` injected as `src_install` and terminate with "TODO" message at install stage?